hledger-lib/Text/Tabular/AsciiWide.hs

```diff
@@ -86,20 +92,30 @@ renderTableB :: TableOpts       -- ^ Options controlling Table rendering
              -> (a -> Cell)   -- ^ Function determining the string and width of a cell
              -> Table rh ch a
              -> Builder
-renderTableB topts@TableOpts{prettyTable=pretty, tableBorders=borders} fr fc f (Table rh ch cells) =
+renderTableB topts fr fc f = renderTableB' topts (fmap fc) (\(rh, as) -> (fr rh, fmap f as))
+
+-- | A version of renderTable that operates on rows
+renderTableB' :: TableOpts      -- ^ Options controlling Table rendering
```

Comment:
I think we should use a more understandable name here, something like `renderTableByRowsB'`.

Reply:
And perhaps a more detailed haddock.
A version of renderTableB which uses a different rendering function for each row.